.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1031 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.vs` should be put in `.gitconfig`. It is user dependent, like `*suo` and `*.user` files.